Examples/Python/tests/test_examples.py

```diff
@@ -62,7 +62,15 @@ def assert_entries(root_file, tree_name, exp):
     rf = ROOT.TFile.Open(str(root_file))
     keys = [k.GetName() for k in rf.GetListOfKeys()]
     assert tree_name in keys
-    assert rf.Get(tree_name).GetEntries() == exp, f"{root_file}:{tree_name}"
+    if non_zero:
+        assert rf.Get(tree_name).GetEntries() > 0, f"{root_file}:{tree_name}"
+    if exp is not None:
+        assert rf.Get(tree_name).GetEntries() == exp, f"{root_file}:{tree_name}"
+
+
+def assert_has_entries(root_file, tree_name):
+    __tracebackhide__ = True
```

Comment:
huch never seen that python magic

Reply:
I think this is pytest specific. It hides it from the trace in case an assertion failure happens. It's just cleaner to find out what's happening.

Reply:
that is actually really cool. I avoided helper assert functions because of that in the past